CMakeLists.txt

```diff
@@ -371,6 +371,12 @@ ENDIF()
 # Create a config header for client application.
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/cmake/data/libraw_config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/libraw_config.h)
 
+# Put the include dirs which are in the source or build tree
+# before all other include dirs, so the headers in the sources
+# are prefered over the already installed ones
+# since cmake 2.4.1
```

Comment:
I don't think we need this line. 2.4 is really old, so this doesn't add any useful information in practice.